app/scripts/controllers/detect-tokens.test.js

```diff
@@ -246,23 +246,7 @@ describe('DetectTokensController', function () {
       onPreferencesStateChange: preferences.store.subscribe.bind(
         preferences.store,
       ),
-      onNetworkStateChange: (cb) =>
-        networkControllerMessenger.subscribe(
-          'NetworkController:networkDidChange',
-          () => {
-            const networkState = network.store.getState();
-            const modifiedNetworkState = {
-              ...networkState,
-              providerConfig: {
-                ...networkState.providerConfig,
-                chainId: convertHexToDecimal(
-                  networkState.providerConfig.chainId,
-                ),
-              },
-            };
-            return cb(modifiedNetworkState);
-          },
-        ),
+      onNetworkStateChange: network.store.subscribe.bind(network.store),
```

Comment:
The original motivation for this change was to stop converting this chain ID to decimal since we don't need to do this. However, there's no real reason why we need to listen for `networkDidChange` in these tests. We probably don't even need a real NetworkController here, but removing that would be a bigger change.